src/actions/types.js

```diff
@@ -98,7 +98,7 @@ export type panelPositionType = "start" | "end";
 
 type UIAction =
   {
-    type: "TOGGLE_FILE_SEARCH",
+    type: "SET_FILE_SEARCH",
```

Comment:
I changed the name from `TOGGLE_FILE_SEARCH ` to `SET_FILE_SEARCH ` because this action involves toggling and closing the file search bar.